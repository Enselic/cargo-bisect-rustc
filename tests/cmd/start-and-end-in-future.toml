bin.name = "cargo-bisect-rustc"
args = "--start 4000-01-01 --end 9999-01-01"
status = "failed"
