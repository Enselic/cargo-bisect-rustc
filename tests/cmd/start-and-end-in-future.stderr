ERROR: start date should be on or before current date, got start date request: 4000-01-01 and current date is [..]
